src/ResourceManager/Sql/Commands.Sql/Auditing/AuditingHelpMessages.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace Microsoft.Azure.Commands.Sql.Auditing
```

Comment:
@ranisha2 Please add a description of these changes to your changelog

Reply:
@markcowl 
I've updated the documentation. Is this needed?

Reply:
@markcowl Updated.